src/components/pages/doc-extensions/what-is-xk6/what-is-xk6.view.js

```diff
@@ -0,0 +1,34 @@
+import { Heading } from 'components/shared/heading';
+import { Link } from 'gatsby';
+import React from 'react';
+
+import styles from './what-is-xk6.module.scss';
+
+export const WhatIsXk6 = () => (
+  <section className={`container ${styles.container}`}>
+    <Heading tag={'h2'} size={'lg'} className={styles.title}>
+      What is xk6?
+    </Heading>
+
+    <p className={styles.description}>
+      <Link to="https://github.com/grafana/xk6/" className="link">
+        xk6
+      </Link>{' '}
+      is the command-line tool and framework inspired by{' '}
+      <Link to="https://github.com/caddyserver/xcaddy/" className="link">
+        xcaddy
```

Comment:
This article is the intro/homepage. I suggest removing the mention to the xcaddy project.